webapp/channels/src/plugins/channel_header_plug/channel_header_plug.test.tsx

```diff
@@ -61,11 +61,11 @@ describe('plugins/ChannelHeaderPlug', () => {
                 shouldShowAppBar={false}
             />,
         );
-        expect(wrapper).toMatchSnapshot();
+        expect(asFragment()).toMatchSnapshot();
```

Comment:
Many of the snapshots we are doing right now only show the final result (the button in place) but it doesn't show anything regarding the overlay. 0/5 if we should do something extra about that. For example...

```
expect('tooltip text').not.toBeInDocument();
userEvent.hover(icon);
expect('tooltip text').toBeInDocument();
```

Or if we prefer snapshot testing:
```
const {asFrament} = renderWithContext(...);
userEvent.hover(icon);
expect(asFragment()).toMatchSnapshot();
```

Reply:
Addressed in https://github.com/mattermost/mattermost/pull/26784